pkg/registry/apis/preferences/stars.go

```diff
@@ -17,7 +16,7 @@ import (
 var _ grafanarest.Storage = (*starStorage)(nil)
 
 type starStorage struct {
-	store grafanarest.Storage
```

Comment:
I found the much better way to wrap the store.... so we *ONLY* override the List function rather than all of them!